tests/infra_test.go

```diff
@@ -366,6 +368,53 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 		)
 	})
 
+	Describe("tls configuration", func() {
+
+		DescribeTable("should be propagated", func(labelSelector string, featureGate string) {
+			cipher := tls.CipherSuites()[0]
+			kvConfig := util.GetCurrentKv(virtClient).Spec.Configuration.DeepCopy()
+			kvConfig.TLSConfiguration = &v1.TLSConfiguration{
+				MinTLSVersion: v1.VersionTLS12,
+				Ciphers:       []string{cipher.Name},
+			}
+			tests.UpdateKubeVirtConfigValueAndWait(*kvConfig)
+			newKv := util.GetCurrentKv(virtClient)
+			Expect(newKv.Spec.Configuration.TLSConfiguration.MinTLSVersion).To(BeEquivalentTo(v1.VersionTLS12))
+			Expect(newKv.Spec.Configuration.TLSConfiguration.Ciphers).To(BeEquivalentTo([]string{cipher.Name}))
+
+			if featureGate != "" {
+				tests.EnableFeatureGate(featureGate)
+			}
+
+			tests.WaitExportProxyReady()
```

Comment:
Should we always wait for the `virt-exportproxy` deployment to be available? even when we don't enable the `VMExport` FG? if this check valid only when enabling the FG, you can move it to the `if featureGate != "" {` condition block

Reply:
Thanks! You're right! I forgot to move the `WaitExportProxyReady()` inside the `if`